tests/test_core.py

```diff
@@ -7121,7 +7121,7 @@ class std_string {
   std::cout << txtTestString.data() << std::endl;
   return 0;
 }
-      ''', '''std_string(const char* s) 
+      ''', '''std_string(const char* s)
```

Comment:
the test failure may be due to this change?

Reply:
Ahh whups, my editor was set to trim whitespace at EOL.  Reverted that line.